### ISSUE TITLE
'Strong Legs' Living Armour Upgrade interferes with Elevators 

### ISSUE THREAD

**Issue by wrincewind** [WayofTime/BloodMagic#1738]
#### Issue Description:
... and possibly with other mods that look out for players jumping. 
Basically, if you use an elevator from the 'elevatorid' mod for 1.16, you jump to go up and sneak to go down. The 'strong legs' enhancement seems to override the usual jumping, so the elevator's not detecting that the player wants to go up.

I know that there's a workaround (hold 'sneak' while using the elevator) but that sends you downstairs 1 floor and is just kind of inconvenient. Plus, I thought i'd report this so others could find this workaround too if they come looking. :p 

#### What happens:
When you jump on an elevator block, you are not teleported upwards as usual.

#### What you expected to happen:
the elevator to function as it does without living armour equipped.

#### Steps to reproduce:

1.  Train up Strong Legs on Living Armour. (I had level V)
2.  place 2 elevator blocks, one above the other
3.  jump on the elevator block to try and go up.

____
#### Affected Versions (Do *not* use "latest"):

- BloodMagic: 3.0.3-8
- Minecraft: 1.16.4
- Forge: 35.1.13
- Elevatorid - 1.16.4-1.7.8
- Modpack: FTB Presents Direwolf20 for 1.16, v 1.2.0

**Comment by WayofTime** [comment]
Wouldn't be surprised if this is due to the player no longer being on the ground when jumping... Could you see if this also happens with a high jump boost potion level?

**Comment by wrincewind** [comment]
Yep, this happens with Jump Boost 2 - it's gotta be on the elevator-guy's side, then. I'll go poke them.

**Comment by wrincewind** [comment]
Interestingly, if you very quickly tap jump (don't hold it), the elevator functions as expected. :P
